### PR TITLE
[LLT-5504] Force nc client to quit and close connection once server-side connection is closed

### PR DESCRIPTION
**Pull request by Jauler**
### Problem
Under "normal" circumstances closing connection from server side does not trigger `nc` to close TCP connection from client-side, which can be observed in the following screenshot:
2024-08-12_14-02-22_grim

In `test_mesh_firewall` tests there is expectation that `nc` will close connections from both sides once at least once side sends `FIN`. To match these expectations `-d` flag has been added, which tells `nc` to not read from stdin, and as a result close client-side connection immediatelly if server-side is closed.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Review by mathiaspeters — DISMISSED** [review]
+1

**Comment by lcruz99** [comment]
+1

**Review by gytsto — APPROVED** [review]
+1

File: nat-lab/tests/test_mesh_firewall.py [hunk 1]
```diff
@@ -601,8 +604,8 @@ async def conntrack_on_stdout(stdout: str) -> None:
             connected_event,
         )
 
-        output_notifier.notify_output("FIN_WAIT", last_ack_event)
-        output_notifier.notify_output("CLOSE_WAIT", time_wait_event)
+        output_notifier.notify_output("LAST_ACK", last_ack_event)
+        output_notifier.notify_output("TIME_WAIT", time_wait_event)
```
Comment (mathiaspeters) [inline nat-lab/tests/test_mesh_firewall.py:608]: I'm curious about the reason for this change. I'm not against it, I would just like to hear the `why` behind it

**Pull request merged**